### PR TITLE
Mock server/antd

### PR DESCRIPTION
**Pull request by 404kacper**
PR prior to setting up mock server. Antd added along with babbel plugin and react-app-rewired, also changed structure of context folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,15 @@
...
+    "babel-plugin-import": "^1.12.0",
     "babel-polyfill": "^6.26.0",
     "classnames": "^2.2.6",
+    "customize-cra": "^0.5.0",
     "lodash": "^4.17.11",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
+    "react-app-rewired": "^2.1.3",
     "react-dom": "^16.8.6",
     "react-scripts": "^3.1.0",
```
Comment (vinodh99) [inline package.json]: Can we remove this package as we are using react-app-rewired?

File: src/App/context/feature/FeatureState.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { useReducer, useCallback } from 'react';
+import FeatureContext from './featureContext';
```
Comment (vinodh99) [inline src/App/context/feature/FeatureState.js]: Minor, let's have a new line between library imports and local imports just to differentiate and for better readability. We shall have code style documented so we don't lose track of these minor issues for anyone new to the project.

Comment (404kacper) [reply src/App/context/feature/FeatureState.js]: Yeah that's what I would usually do, I've been in quite a hurry writing this line sorry for ommiting that.

File: package.json (hunk 1 above)
Comment (404kacper) [reply package.json]: I don't quite get what you're saying, without this package modules necessary for react-app-rewired won't be installed therefore the app won't run. Could you expand your answear a bit more?

Comment (vinodh99) [reply package.json]: I mean to remove react-scripts as we are no longer using it

**Pull request merged**